5 lesson redux/src/app/actions/userActions.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+
+export function fetchUsers() {
+    return function(dispatch) {
+        dispatch({type: 'FETCH_USERS'});
+
+        axios.get("http://jsonplaceholder.typicode.com/users")
+        .then((response) => {
+            const data = response.data;
+            dispatch({
+            type: 'FETCH_USERLIST_FULFILLED', 
+            payload: {
```

Comment:
```suggestion
            payload: data
```
не надо оборачивать в объект:)